python/py-reportlab/Portfile

```diff
@@ -21,11 +21,19 @@ long_description    The ReportLab Toolkit is a library for programatically \
 
 homepage            https://www.reportlab.com/software/opensource/rl-toolkit/
 
-checksums           rmd160  58327ab1dd63af72a225a9ae6a262a5ba12a8c10 \
-                    sha256  68f9324000cfc5570b5a59a92306691b5d655078a399f20bc72c2581fe903261 \
-                    size    4511585
+checksums           rmd160  12c13059744117c384ce593dae676a2197dcff18 \
+                    sha256  f0c4b47b012d893b0b9f5703cf6f01b5593714a3fc1e7dc73efbbfe26bb7e16a \
+                    size    4501397
 
 if {${name} ne ${subport}} {
+    if {${python.version} <= 27} {
+        version     3.5.68
```

Comment:
did the PY27 port ever install for this version (but just didn't work?). If so, you will need to increase the "epoch" for user to see this "update". If you're just restoring it/fixing a build error this is okay.

Reply:
No, it hasn't worked since the port was at 3.5.46, so I believe we don't need to bump the epoch.